index.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
 				'ava/no-statement-after-end': 'error',
 				'ava/no-todo-implementation': 'error',
 				'ava/no-todo-test': 'warn',
+				'ava/no-typeerror-with-notthrows': 'error',
```

Comment:
This rule name doesn't make sense as it's not just about `TypeError`, but any error.

Reply:
@sindresorhus  how about just `no-error-with-notthrows` ?